python/pyspark/sql/functions.py

```diff
@@ -206,17 +227,20 @@ def _():
 @since(1.3)
 def approxCountDistinct(col, rsd=None):
     """
-    .. note:: Deprecated in 2.1, use approx_count_distinct instead.
+    .. note:: Deprecated in 2.1, use :func:`approx_count_distinct` instead.
     """
     return approx_count_distinct(col, rsd)
 
 
 @since(2.1)
 def approx_count_distinct(col, rsd=None):
-    """Returns a new :class:`Column` for approximate distinct count of ``col``.
+    """Returns a new :class:`Column` for approximate distinct count of column `col`.
```

Comment:
Sounds like it also needs leading `Aggregate function: ` doc - https://github.com/apache/spark/blob/3a840048ed3501e06260b7c5df18cc0bbdb1505c/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L225.